config.default

```diff
@@ -152,6 +152,11 @@ overwriteMpasClimatology = False
 #   'bilinear', 'neareststod' (nearest neighbor) or 'conserve'
 mpasInterpolationMethod = bilinear
 
+# the number of years per cached climatology file.  These cached files are
+# aggregated together to create annual climatologies, for example, when
+# computing the MOC.
```

Comment:
I use MPAS-Analysis and I want to access the data for further, separate analysis.  For instance, do we have easy accessor methods for the cached datasets?  

Maybe this isn't an important use case but one could imagine wanting to access the cached information in some clean way.